Documentation/Administrator_functions.md

```diff
@@ -0,0 +1,28 @@
+##  TIN - Administrator functions documentation
+
+### Available functions for administrator
+
+#### Connect
+_connect <"server address:port">_
+Connects to server on the specified address (address provided with port). 
+
+#### Sending process
+_send_process <"name"> <"path to file">_
+User specifies the <"path to the file"> which will be sent to the server. The file should be on the user's machine.
+The argument <"name"> is the process's name on the server.
```

Comment:
That's nice.
Only one note to detail: We need to remember that only admin has access to name of process given by user. So in admin we should have map<string, string>, where key is a name g. You're creating unique UUID, that is inserted as value for given key(we have functions for creating this in C++ for sure). So only UUID is send to the server(and later to node). Later, you're receiving UUID, and doing mapping to name.

So maybe not a map, but rather set<tuple<string, string>>. :) To support fast find in every case. 
So basically user could write "processss1", but node and server sees only unique identifier given by node.